src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -336,20 +336,21 @@ abstract class RefChecks extends Transform {
           analyzer.foundReqMsg(pair.lowClassBound, pair.highClassBound)
         )
 
+        val indent = "  "
         def overrideErrorMsg(msg: String): String = {
           val isConcreteOverAbstract =
             (other.owner isSubClass member.owner) && other.isDeferred && !member.isDeferred
           val addendum =
             if (isConcreteOverAbstract)
-              ";\n (Note that %s is abstract,\n  and is therefore overridden by concrete %s)".format(
+              ";\n${indent}(note that %s is abstract,\n${indent}and is therefore overridden by concrete %s)".format(
```

Comment:
I'm sorry `-Xlint` is not turned on for the build to catch the missing interpolator. Probably a good moment to get rid of the format call, since it's not doing much formatting, so `s""` is good enough. But if you choose `f`, the newline should be `f"%n"` for platform line separator, sigh.

Reply:
Interesting. I didn't catch that locally because neg/t5358.scala passed.
I'll fix this.